src/main/java/org/iitbact/erp/services/ApiValidationService.java

```diff
@@ -23,5 +23,22 @@ public String verifyFirebaseIdToken(BaseRequest request) {
 		}
 		*/
 	}
+	public String verifyFirebaseIdToken(String authToken) {
```

Comment:
should this be _get_FirebaseIdToken? And is this a secret string or just an app id?